app/views/tags/_table.html.erb

```diff
@@ -1,25 +1,32 @@
 <% cache ['tags-table', @all_columns, @tags] do %>
```

Comment:
Should we bust this cache key when the order is changed?

Reply:
Updated the `Sortable` concern so it's now updating the records' `updated_at` values when we change the order. Previously it wasn't being updated because we're using `update_all`